ext/standard/streamsfuncs.c

```diff
@@ -827,9 +827,9 @@ PHP_FUNCTION(stream_select)
 	retval = php_select(max_fd+1, &rfds, &wfds, &efds, tv_p);
 
 	if (retval == -1) {
-		php_error_docref(NULL, E_WARNING, "unable to select [%d]: %s (max_fd=%d)",
+		zend_value_error("Unable to select [%d]: %s (max_fd=%d)",
```

Comment:
I am a little bit unsure about this one..

Reply:
Yeah, this shouldn't be ValueError.

Reply:
Fixed.